src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -2484,15 +2484,20 @@ public Document GetRevision(DocumentsOperationContext context, string changeVect
             }
         }
 
-        public (int ActualSize, int AllocatedSize, bool IsCompressed)? GetRevisionMetrics(DocumentsOperationContext context, string changeVector)
+        public bool GetRevisionMetrics(DocumentsOperationContext context, string changeVector,out (int ActualSize, int AllocatedSize, bool IsCompressed) metrics)
```

Comment:
`GetRevisionMetrics` -> `TryGetRevisionMetrics`

Reply:
or even better.. let's keep this identical to the doc logic

Reply:
Done